docs/tutorials/batching-simulations.md

```diff
@@ -123,24 +122,24 @@ So we want to run a total of `11 * 11 = 121` simulations. Let's compare how long
 <!-- skip: start -->
 
 ```pycon
->>> %timeit run_unbatched('cpu')
-1.78 s ± 184 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
->>> %timeit run_batched('cpu')
-284 ms ± 129 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
+>>> %timeit run_unbatched()
+119 ms ± 9.18 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
+>>> %timeit run_batched()
+56.5 ms ± 539 µs per loop (mean ± std. dev. of 7 runs, 1 loop each)
 ```
 
-Even with this simple example, we gain a **factor x6** in speedup just from batching.
+Even with this simple example, we gain a **factor x2** in speedup just from batching.
 
 The result is even more striking on GPU[^2]:
 [^2]: NVIDIA GeForce RTX 4090.
 
 ```pycon
->>> %timeit run_unbatched('cuda')
-1.51 s ± 1.98 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
->>> %timeit run_batched('cuda')
-20.1 ms ± 78.7 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
+>>> %timeit run_unbatched()
+439 ms ± 692 µs per loop (mean ± std. dev. of 7 runs, 1 loop each)
+>>> %timeit run_batched()
+6.29 ms ± 160 µs per loop (mean ± std. dev. of 7 runs, 1 loop each)
 ```
 
-On the GPU, because we save costly data transfers with the CPU and do N-D matrices multiplications, we gain a **factor x75** in speedup!
+On the GPU, because we save costly data transfers with the CPU and do N-D matrices multiplications, we gain a **factor x70** in speedup!
```

Comment:
There is not much data transfer involved that we remove, only latency. This phrase should be corrected or removed